Makefile

```diff
@@ -23,9 +23,17 @@ export DOCKER_IMAGE_TAG?=latest
 GOPATH:=$(shell go env GOPATH)
 BINPATH:=$(abspath ./bin)
 SUBMODULES=_submodules
+UID:=$(shell id -u)
+
+FIRECRACKER_DIR=$(SUBMODULES)/firecracker
+FIRECRACKER_BIN=$(FIRECRACKER_DIR)/target/x86_64-unknown-linux-musl/release/firecracker
+JAILER_BIN=$(FIRECRACKER_DIR)/target/x86_64-unknown-linux-musl/release/jailer
```

Comment:
Can we make these paths architecture independent without too much work? At some point we'll need arm64, at least.

Reply:
Yeah, for now I just updated to use an explicit `FIRECRACKER_TARGET` var that is used in the Makefile and also passed to the Dockerfile as a build-arg.